packages/reakit/src/Toolbar/__examples__/ToolbarWithMenu/__tests__/index-test.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import { render, press, click } from "reakit-test-utils";
+import ToolbarWithMenu from "..";
+
+describe("<ToolbarWithMenu />", () => {
```

Comment:
Maybe we should have a lint rule for that, but, except for very specific cases, I try not to nest test suites with `describe` to make things simpler as the file itself already provides the context.
```suggestion
```

Reply:
Ah I forgot 😓 ,  that's my habit of sticking to this style, let me refactor them to adhere to rest of the codebase 🙏🏻